src/app/home-page/home-page.component.html

```diff
@@ -0,0 +1,22 @@
+<div id="HomePageInfo">
```

Comment:
Don't know how to feel about `id` parameter. If you are planning to use it once, technically it's not a mistake. I think to use `class` would be better in this case. `id` would work perfectly with tables, images or other stuff for unique styling.